computation/src/main/java/com/powsybl/computation/ComputationExceptionBuilder.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.computation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class ComputationExceptionBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ComputationExceptionBuilder.class);
+
+    private final Map<String, String> outMsgByLogFileName = new HashMap<>();
+    private final Map<String, String> errMsgByLogFileName = new HashMap<>();
+    private final Map<String, byte[]> bytesByFileName = new HashMap<>();
+    private final List<Exception> exceptions = new ArrayList<>();
+
+    public ComputationExceptionBuilder(Exception exception) {
```

Comment:
Do we need another constructor that takes an array or a list?

Reply:
We could add this later. If someone really needs it.